feathr-example/scripts/poolupdate.sh

```diff
@@ -0,0 +1,32 @@
+subscription_id="<SUBSCRIPTION_ID>"
```

Comment:
Where is this script used?

Reply:
Based on offline chat with @daxsorbito : this script was used as a workaround for a bug related to spark configuration in the source feathr repo. The bug was fixed in the source repo; So this script can be removed.